Assets/MRTK/Services/InputSystem/GazeProvider.cs

```diff
@@ -440,6 +440,15 @@ protected override void OnDisable()
             }
         }
 
+        /// <inheritdoc />
+        private void OnDestroy()
+        {
+            if (GazeCursor != null && !GazeCursor.Equals(null))
```

Comment:
Can you leave a comment about why we both check against null and also against null?